### PR TITLE
src/editor.rs: Fix comments

### PR DESCRIPTION
**Pull request by ldang0**

### CODE REVIEW

File: src/editor.rs [hunk 1]
```diff
@@ -137,7 +137,7 @@ impl<'a> Editor<'a> {
     /// # Errors
     ///
     /// Will return `Err` if an error occur when enabling termios raw mode, creating the signal hook
```
Comment (ilai-deutel) [inline src/editor.rs]: Just noticed this typo, can you also fix it?

```suggestion
    /// Will return `Err` if an error occurs when enabling termios raw mode, creating the signal hook
```

File: src/editor.rs [hunk 2]
```diff
@@ -137,7 +137,7 @@ impl<'a> Editor<'a> {
...
-    /// or when obtaining the terminal window size.
```
Comment (ilai-deutel) [inline src/editor.rs]: `new` will return `Err` if an error occurs when obtaining the terminal window size initially (`editor.update_window_size()?`)

Comment (ldang0) [reply src/editor.rs]: Oh, this make more sense :-)

Comment (ldang0) [reply src/editor.rs]: Oh, I see now.  Thank you.

**Pull request merged**